compositor_render/src/scene.rs

```diff
@@ -92,7 +91,7 @@ pub(crate) enum NodeParams {
     InputStream(InputId),
     Shader(ShaderComponentParams, Arc<Shader>),
     Web(Vec<ComponentId>, Arc<WebRenderer>),
-    Image(ImageRenderParams),
+    Image(StatefulImageComponent),
```

Comment:
It feels like a very bad idea, you are forcing here cloning statefull components, which can easily lead to cloning that by mistake and somehow operating on cloned state by mistake